mountpoint-s3/src/data_cache/express_data_cache.rs

```diff
@@ -149,6 +172,8 @@ mod tests {
     use mountpoint_s3_client::mock_client::{MockClient, MockClientConfig};
     use mountpoint_s3_client::types::ETag;
 
+    const EXPRESS_CACHE_MAX_OBJECT_SIZE: usize = 1024 * 1024;
```

Comment:
This should definitely be a constant somewhere as it's reused across files

Reply:
factored out to the next PR